clients/python/tests/test_cli_smoke.py

```
@@ -0,0 +1,166 @@
+            pass
+
+
+class TestCliSmoke:
+    def test_golden_file_output(self, cli_server):
+        result = subprocess.run(
+            [sys.executable, "-m", "anna", "--config", cli_server, "cli", INPUT_FILE],
+            capture_output=True, text=True, timeout=60,
+            cwd=os.path.join(REPO_ROOT, "clients", "python"),
+        )
+
+        actual = [normalize_set_line(l.rstrip()) for l in result.stdout.splitlines() if l.strip()]
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)